package.json

```diff
@@ -4,7 +4,7 @@
     "prepare": "husky install"
   },
   "devDependencies": {
-    "husky": "^8.0.2",
+    "husky": "^4.3.8",
```

Comment:
curious how this change happened lol - does everything work properly with `8.0.2`?

Reply:
it does - one day I was having issues with lint-staged or something (don't quite remember) so I googled and they told me to reinstall husky so that's probably how that happened